cimgui/cimgui.h

```diff
@@ -457,10 +472,12 @@ CIMGUI_API void             igGetWindowSize(struct ImVec2* out);
 CIMGUI_API float            igGetWindowWidth();
 CIMGUI_API float            igGetWindowHeight();
 CIMGUI_API bool             igIsWindowCollapsed();
+CIMGUI_API bool             igIsWindowAppearing();
 CIMGUI_API void             igSetWindowFontScale(float scale);
 
-CIMGUI_API void             igSetNextWindowPos(CONST struct ImVec2 pos, ImGuiCond cond);
-CIMGUI_API void             igSetNextWindowPosCenter(ImGuiCond cond);
+CIMGUI_API void             igSetNextWindowPos(CONST struct ImVec2 pos, ImGuiCond cond, CONST struct ImVec2 pivot);
+//Is going to be obsolete
+//CIMGUI_API void             igSetNextWindowPosCenter(ImGuiCond cond);
```

Comment:
Let us not use comments for obsoletion. Please remove the code instead of commenting it